elements/pouch-db/src/pouch-db.js

```diff
@@ -0,0 +1,168 @@
+/**
+ * Copyright 2019
+ * @license Apache-2.0, see License.md for full text.
+ */
+import { html, PolymerElement } from "@polymer/polymer/polymer-element.js";
+
+//WHERE SHOULD THIS GO???
+import "./pouchdb.min.js";
+
+// register globally so we can make sure there is only one
+window.PouchDb = window.PouchDb || {};
+// request if this exists. This helps invoke the element existing in the dom
+// as well as that there is only one of them. That way we can ensure everything
+// is rendered through the same pouch-db element, making it a singleton.
+window.PouchDb.requestAvailability = () => {
+  // if there is no single instance, generate one and append it to end of the document
+  if (!window.PouchDb.instance) {
+    window.PouchDb.instance = document.createElement("pouch-db");
+    document.body.appendChild(window.PouchDb.instance);
+  }
+  return window.PouchDb.instance;
+};
+
+/**
+ * `pouch-db`
+ * `read and write localized data elements`
+ *
+ * @microcopy - language worth noting:
+ *  -
+ *
+ * @customElement
+ * @polymer
+ * @demo demo/index.html
+ */
+class PouchDb extends PolymerElement {
+  /* REQUIRED FOR TOOLING DO NOT TOUCH */
+
+  /**
+   * Store the tag name to make it easier to obtain directly.
+   * @notice function name must be here for tooling to operate correctly
+   */
+  static get tag() {
+    return "pouch-db";
+  }
+  /**
+   * life cycle, element is afixed to the DOM
+   */
+  connectedCallback() {
+    super.connectedCallback();
+    window.addEventListener("pouch-db-hide", this.hidePouchDb.bind(this));
+    window.addEventListener("pouch-db-show", this.showPouchDb.bind(this));
+    window.addEventListener(
+      "user-engagement",
+      this.userEngagmentFunction.bind(this)
+    );
+  }
+
+  userEngagmentFunction(e) {
+    var db = new PouchDB("xapistatements");
+    var remoteCouch = false;
+    ///var remoteCouch = 'http://35.164.8.64:3000/todos';
+
+    function createStatement(quizNameAdded, quizPassed, callback) {
```

Comment:
scope these to the element tho i'm sure your happy they were just working initially :p

Reply:
done.